From 9015e6678e61c413939ad83589542712950c93ff Mon Sep 17 00:00:00 2001
From: David Mercado <david.mercado@windriver.com>
Date: Tue, 19 Nov 2013 11:26:33 -0500
Subject: [PATCH 128/175] LSI AXM55XX: Add clock-frequency entry in DTS file

git.yoctoproject.org/git/linux-yocto-3.10
commit eea914b76e8602a9da4dcb91707bd844aeccf3dc standard/axxia/base.

Add clock-frequency placeholder to axm55xx.dts file.

Signed-off-by: David Mercado <david.mercado@windriver.com>
---
 arch/arm/boot/dts/axm55xx.dts |   20 ++++++++++++++++++--
 1 files changed, 18 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/axm55xx.dts b/arch/arm/boot/dts/axm55xx.dts
index aac791d..20d56cd 100644
--- a/arch/arm/boot/dts/axm55xx.dts
+++ b/arch/arm/boot/dts/axm55xx.dts
@@ -1,5 +1,5 @@
 /*
- * arch/arm/boot/dts/axm5500.dts
+ * arch/arm/boot/dts/axm55xx.dts
  *
  * Copyright (C) 2012 LSI
  *
@@ -46,6 +46,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <0>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@1 {
@@ -53,6 +54,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <1>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@2 {
@@ -60,6 +62,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <2>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@3 {
@@ -67,6 +70,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <3>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@4 {
@@ -74,6 +78,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <4>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@5 {
@@ -81,6 +86,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <5>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@6 {
@@ -88,13 +94,15 @@
 			compatible = "arm,cortex-a15";
 			reg = <6>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@7 {
-			device_type = "cpua";
+			device_type = "cpu";
 			compatible = "arm,cortex-a15";
 			reg = <7>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@8 {
@@ -102,6 +110,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <8>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@9 {
@@ -109,6 +118,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <9>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@10 {
@@ -116,6 +126,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <10>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@11 {
@@ -123,6 +134,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <11>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@12 {
@@ -130,6 +142,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <12>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@13 {
@@ -137,6 +150,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <13>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@14 {
@@ -144,6 +158,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <14>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 
 		cpu@15 {
@@ -151,6 +166,7 @@
 			compatible = "arm,cortex-a15";
 			reg = <15>;
 			cpu-release-addr = <0>; // Fixed by the boot loader
+			clock-frequency= <0>;   // Placeholder
 		};
 	};
 
-- 
1.7.1

